src/docs/community/migration.mdx

```diff
@@ -0,0 +1,242 @@
+
+## @brightlayer-ui/icons-mui
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/icons-svg
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/mapbox
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/ng-progress-icons
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/prettier-config
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/publish
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/react-auth-shared
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+
+## @brightlayer-ui/react-auth-workflow
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/react-components
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   For using App Bar in app, you can use `import { AppBar } from '@brightlayer-ui/react-components';`
+
+## @brightlayer-ui/react-native-progress-icons
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/react-native-template-authentication
+
+-   `pxb` preface changed to `blui`.
+-   You can use these templates with the Brightlayer UI CLI:
+
+-   ```sh
+      npx -p @brightlayer-ui/cli blui new react-native --cli=rnc --template=authentication
+    ```
+
+## @brightlayer-ui/react-native-template-authentication-typescript
+
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/react-native-template-blank
+
+-   `pxb` preface changed to `blui`.
+-   npx -p @brightlayer-ui/cli blui new react-native --cli=rnc --template=blank
+
+## @brightlayer-ui/react-native-template-blank-typescript
+
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/react-native-template-routing
+
+-   `pxb` preface changed to `blui`.
+-   You can use these templates with the Brightlayer UI CLI:
+
+-   ```sh
+      npx -p @brightlayer-ui/cli blui new react-native --cli=rnc --template=routing
+    ```
+
+## @brightlayer-ui/react-native-template-routing-typescript
+
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/react-native-themes
+
+-   Changed theme import from `PXBThemes` to `BLUIThemes`.
+
+## @brightlayer-ui/react-native-vector-icons
+
+-   `pxb` preface changed to `blui`.
+-   Changed Icon import from `PXBIcon` to `BLUIIcon`.
+-   Changed `RNPXBVectorIcons` to `RNBLUIVectorIcons`.
+
+## @brightlayer-ui/react-progress-icons
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   `pxb` preface changed to `blui`.
+
+## @brightlayer-ui/react-themes
+
+-   Changed package namespace from `@pxblue` to `@brightlayer-ui`.
+-   Changed import themes from `PXBlueThemes` to `BLUIThemes`.
```

Comment:
There is nothing in the themes package called PXBlueThemes...this is only relevant if someone is importing the themes like `import * as Themes from '@pxblue/react-themes` in which case, this is not a required change for upgrading...they can call that default import anything they want and it will work.